crates/common/consensus/src/deneb/beacon_state.rs

```diff
@@ -81,11 +82,36 @@ use crate::{
     withdrawal::Withdrawal,
 };
 
+fn serialize_u8_as_quoted_list<S>(
+    value: &VariableList<u8, U1099511627776>,
+    serializer: S,
+) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    let string_vec: Vec<String> = value.iter().map(|v| v.to_string()).collect();
+    string_vec.serialize(serializer)
+}
+fn quoted_u8_var_list<'de, D>(deserializer: D) -> Result<VariableList<u8, U1099511627776>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let string_vec: Vec<String> = Vec::deserialize(deserializer)?;
+    let bytes = string_vec
+        .into_iter()
+        .map(|s| s.parse::<u8>().map_err(serde::de::Error::custom))
+        .collect::<Result<Vec<_>, _>>()?;
+    VariableList::new(bytes)
+        .map_err(|_| serde::de::Error::custom("Cannot create VariableList from bytes"))
```

Comment:
```suggestion
        .map_err(|err| serde::de::Error::custom(format!("Cannot create VariableList from bytes {err:?}")))
```

are we able to pass up this error?